lit/docs/auth/roles.lit

```diff
@@ -43,77 +104,185 @@ Concourse comes with five roles: Concourse Admin, Team Owner, Team Member, Pipel
-    \table-row{                   }{Resume Pipeline         }{✓    }{✓           }{✓            }{✓                       }{✘            }
-    \table-row{                   }{Reorder Pipeline        }{✓    }{✓           }{✓            }{✘                       }{✘            }
-    \table-row{Pipeline Page      }{View                    }{✓    }{✓           }{✓            }{✓                       }{✓            }
-    \table-row{                   }{Click to Resource       }{✓    }{✓           }{✓            }{✓                       }{✓            }
-    \table-row{                   }{Click to Build          }{✓    }{✓           }{✓            }{✓                       }{✓            }
-    \table-row{                   }{Click on Group          }{✓    }{✓           }{✓            }{✓                       }{✓            }
-    \table-row{Resource Page      }{View Resource           }{✓    }{✓           }{✓            }{✓                       }{✓            }
-    \table-row{                   }{View Version Details    }{✓    }{✓           }{✓            }{✓                       }{✓            }
-    \table-row{                   }{Pin Version             }{✓    }{✓           }{✓            }{✓                       }{✘            }
-    \table-row{                   }{Paginate (<- ->)        }{✓    }{✓           }{✓            }{✓                       }{✓            }
-    \table-row{Build Page         }{Trigger new Build       }{✓    }{✓           }{✓            }{✓                       }{✘            }
-    \table-row{                   }{View Build              }{✓    }{✓           }{✓            }{✓                       }{✓            }
-    \table-row{                   }{Build Details           }{✓    }{✓           }{✓            }{✓                       }{✓            }
-    \table-row{                   }{Abort Build             }{✓    }{✓           }{✓            }{✓                       }{✘            }
-    \table-row{Job Page           }{View Job Page           }{✓    }{✓           }{✓            }{✓                       }{✓            }
-    \table-row{                   }{Pause Job               }{✓    }{✓           }{✓            }{✓                       }{✘            }
-    \table-row{                   }{Trigger new Build       }{✓    }{✓           }{✓            }{✓                       }{✘            }
-    \table-row{                   }{Build History           }{✓    }{✓           }{✓            }{✓                       }{✓            }
-    \table-row{                   }{Paginate (<- ->)        }{✓    }{✓           }{✓            }{✓                       }{✓            }
-  }
+\section{
+  \title{Configuring RBAC}{configuring-rbac}
+  \warn{Configuring RBAC is \bold{experimental}, and this may change in the future.}
+
+  It is possible to modify the set of permitted actions for a given role, by
```

Comment:
There's a quite subtle behavior that's hard to summarize with one word, but I think using "modify" here should come with a caveat. This sentence suggests to me that I can add or remove roles in a granular way, and someone coming from using GCP or AWS IAM might carry over assumptions that Concourse roles work the same way. With that mental reference point, I would be surprised to discover that in Concourse, it's not true (for example) that I can grant a role to a Team Viewer but not a Pipeline Operator.

I wonder if we can make it somehow clearer that Concourse roles work like a pyramid or a cascade, and _not_ like buckets of roles, like how other systems commonly work. Something like.. "Additional roles can be granted to a user type if they're not included by default; otherwise a role granted by default to a lower-permissioned role type can be "hoisted up" into the subset of roles belonging to a more highly-permissioned user type."

Reply:
Also gonna tag in @matthewpereira for wordsmithing input!